src/tests/functional/plugin/shared/src/behavior/plugin/configuration_tests.cpp

```diff
@@ -107,6 +107,16 @@ TEST_P(CorrectConfigTests, CanUseCache) {
     CommonTestUtils::removeDir("./test_cache");
 }
 
+TEST_P(SetAllowAutoBatchingCorrectConfigTestCheck, SetConfigNoThrow) {
+    ie->SetConfig({ {CONFIG_KEY(ALLOW_AUTO_BATCHING), CONFIG_VALUE(YES)} }, targetDevice);
```

Comment:
this SetConfig should be also wrapped with ASSERT_NO_THROW, also the second one below
https://github.com/openvinotoolkit/openvino/pull/10399/commits/13666d3ce71c8dd4888feae85a2b164c0867a284

Reply:
test cases in CPU plugin is done and in GPU plugin will update soon.